src/cache_cargo_home.rs

```diff
@@ -76,14 +76,10 @@ impl Group {
     }
 
     pub fn last_modified(&self) -> Option<DateTime<Utc>> {
-        let mut result = None;
-        for fingerprint in self.entries.values() {
-            result = match (result, fingerprint.modified()) {
-                (None, modified) | (modified, None) => modified,
-                (Some(a), Some(b)) => Some(std::cmp::max(a, b)),
-            };
-        }
-        result
+        self.entries
+            .values()
+            .filter_map(|fingerprint| fingerprint.modified())
```

Comment:
Actually I think if you run clippy it will suggest that you can write this without building a closure by doing something like `.filter_map(Fingerprint::modified)`.

Reply:
You didn't give any details, but I guess maybe you needed to install the `wasm32-unknown-unknown` target? With `rustup` this would be `rustup install target wasm32-unknown-unknown`.

Reply:
I thought it was something with my recent rustup update, and it is not related to this exact project, but your suggestion about `rustup target add wasm32-unknown-unknown` helped me to run cargo, thanks!